main/main.js

```diff
@@ -21,21 +45,23 @@ crashReporter.start({
 })
 
 if (process.argv.some(arg => arg === '-v' || arg === '--version')) {
-  console.log('Min: ' + app.getVersion())
-  console.log('Chromium: ' + process.versions.chrome)
+  console.log(`Min: ${app.getVersion()}`)
+  console.log(`Chromium: ${process.versions.chrome}`)
   process.exit()
 }
 
+// arg === '--development-mode' -> arg === CMD_DEV_MODE
 let isInstallerRunning = false
 const isDevelopmentMode = process.argv.some(arg => arg === '--development-mode')
 
 function clamp (n, min, max) {
   return Math.max(Math.min(n, max), min)
 }
 
-if (process.platform === 'win32') {
+if (process.platform === PLATFORM_WIN) {
```

Comment:
Isn't `process.platform === PLATFORM_WIN` exactly the same as doing `process.platform === 'win32'` and similars?

Reply:
Yes, but, constant variables be clean and human readable as like Enums  😉✌